src/investments/registered-retirement-income-fund.ts

```diff
@@ -11,56 +11,58 @@ export interface RegisteredRetirementIncomeFund {
 }
 
 export const RRIF: RegisteredRetirementIncomeFund = {
+    // This is based on January 1st, but it is meant to be used with age at the end of the year, so we add one year.
     MIN_WITHDRAWAL_PCT: {
-        50: 0.0250,
-        51: 0.0256,
-        52: 0.0263,
-        53: 0.0270,
-        54: 0.0278,
-        55: 0.0286,
-        56: 0.0294,
-        57: 0.0303,
-        58: 0.0313,
-        59: 0.0323,
-        60: 0.0333,
-        61: 0.0345,
-        62: 0.0357,
-        63: 0.0370,
-        64: 0.0385,
-        65: 0.0400,
-        66: 0.0417,
-        67: 0.0435,
-        68: 0.0455,
-        69: 0.0476,
-        70: 0.0500,
-        71: 0.0528,
-        72: 0.0540,
-        73: 0.0553,
-        74: 0.0567,
-        75: 0.0582,
-        76: 0.0598,
-        77: 0.0617,
-        78: 0.0636,
-        79: 0.0658,
-        80: 0.0682,
-        81: 0.0708,
-        82: 0.0738,
-        83: 0.0771,
-        84: 0.0808,
-        85: 0.0851,
-        86: 0.0899,
-        87: 0.0955,
-        88: 0.1021,
-        89: 0.1099,
-        90: 0.1192,
-        91: 0.1306,
-        92: 0.1449,
-        93: 0.1634,
-        94: 0.1879,
-        95: 0.20,
+        51: 0.0250,
+        52: 0.0256,
+        53: 0.0263,
+        54: 0.0270,
+        55: 0.0278,
+        56: 0.0286,
+        57: 0.0294,
+        58: 0.0303,
+        59: 0.0313,
+        60: 0.0323,
+        61: 0.0333,
+        62: 0.0345,
+        63: 0.0357,
+        64: 0.0370,
+        65: 0.0385,
+        66: 0.0400,
+        67: 0.0417,
+        68: 0.0435,
+        69: 0.0455,
+        70: 0.0476,
+        71: 0.0500,
+        72: 0.0528,
+        73: 0.0540,
+        74: 0.0553,
+        75: 0.0567,
+        76: 0.0582,
+        77: 0.0598,
+        78: 0.0617,
+        79: 0.0636,
+        80: 0.0658,
+        81: 0.0682,
+        82: 0.0708,
+        83: 0.0738,
+        84: 0.0771,
+        85: 0.0808,
+        86: 0.0851,
+        87: 0.0899,
+        88: 0.0955,
+        89: 0.1021,
+        90: 0.1099,
+        91: 0.1192,
+        92: 0.1306,
+        93: 0.1449,
+        94: 0.1634,
+        95: 0.1879,
+        96: 0.20,
     },
 };
 
 export function getMinimumWithdrawalPercentage(age: number): number {
```

Comment:
Je crois que Kevin t'a envoyé dans une mauvaise piste. Ses intentions étaient bonne pour aider, mais nous allons introduire un bug en faisant les choses de cette façon.

Je crois que tax-ca ne doit pas changer, et doit représenter les constantes explicites tel que défini dans les différents site du gouvernement. Si on veut vraiment faire un changement dans tax-ca, selon moi la chose à faire serait peut-être de renommer la fonction ou le paramètre pour préciser que c'est avec l'âge en début d'année qu'on doit faire le calcul. Donc, peut-être quelque chose comme ça

```suggestion
export function getMinimumWithdrawalPercentage(beginningOfYearAge: number): number {
```